### ISSUE TITLE
Analyze timeout

### ISSUE THREAD

**Issue by flankerhqd** [sireum/amandroid#10]
Trying to run reachablility analysis on an 209kb apk got 30min timeout with icc enabled, -Xmx24g, context 1, static clinit disabled.  Some components are analyzed but others timeout.

Test case attached as below:

 http://box.myqsc.com/-04822815

**Comment by lucasbrutschy** [comment]
Does the latest version already include the push? Did you find other sources of timouts? For me, not a single example terminates...

**Comment by fgwei** [comment]
Iucasbrutschy, have you give enough memory? And what kind of apks you try to test?

**Comment by lucasbrutschy** [comment]
Yes. I ran it on real world APKs from the play store, 500KB or so. I want to run it on top apps. For a start, I ran it on joansoft.dailybible w/ options sireum amandroid staging -m 30 -to 2000 but I got

[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!
[CRITICAL]Timeout!
Timeout!

Does not seem to consume a lot of memory. Can I increase the number of used cores?

**Comment by fgwei** [comment]
Could you send me that joansoft.dailybible app to check?

**Comment by lucasbru** [comment]
http://ge.tt/3xpnUi92/v/0?c

Thanks for checking!

**Comment by lucasbru** [comment]
Any news on this?
